src/css/modules/_SectionHeader.scss

```diff
@@ -0,0 +1,12 @@
+.coa_SectionHeader {
+  text-transform: uppercase;
+  text-align: center;
+  &:after {
+    margin: $coa-spacing-medium auto $coa-spacing-xlarge;
+    width: 40px;
```

Comment:
When is it chill to have magic numbers like this? For me, I don't mind too much. Would it be better at all to call them out and assign them as sass vars and throw them at the top of the page? idk. seeing it a lot for widths and heights

Reply:
mmmm, honestly, not too sure.